File: monitorias/Personalizadas - individuales/Fundamentos de programación - Santiago/Condicionales (if, else, switch)/Condicionales.py
### If / else → decisiones simples.
numero = 10
if numero > 5:
    print("El número es mayor que 5")
else:
    print("El número es menor o igual que 5")
    
### If anidado → varias condiciones.
color = "rojo"
if color == "azul":
    print("El color es azul")
elif color == "rojo":
    print("El color es rojo")
else:
    print("El color no es azul ni rojo")

### Ejercicio
edad = int(input("Introduce tu edad: "))

if edad < 18:
    print("Eres menor de edad")
else:
    if edad < 65:
        print("Eres adulto")
    else:
        print("Eres adulto mayor")

File: monitorias/Personalizadas - individuales/Fundamentos de programación - Santiago/Condicionales (if, else, switch)/Condicionales.java
import java.util.Scanner;

public class Condicionales {
    public static void main(String[] args) {
        // Ejemplo de if/else (decisión simple)
        int numero = 10;
        if (numero > 5) {
            System.out.println("El número es mayor que 5");
        } else {
            System.out.println("El número es menor o igual que 5");
        }

        // Ejemplo de if anidado (varias condiciones)
        Scanner sc = new Scanner(System.in);
        System.out.print("Introduce tu edad: ");
        int edad = sc.nextInt();
        if (edad < 18) {
            System.out.println("Eres menor de edad");
        } else {
            if (edad < 65) {
                System.out.println("Eres adulto");
            } else {
                System.out.println("Eres adulto mayor");
            }
        }

        // Ejemplo de switch (alternativa al if múltiple)
        int opcion = 2;
        switch(opcion) {
            case 1:
                System.out.println("Opción 1");
                break;
            case 2:
                System.out.println("Opción 2");
                break;
            case 3:
                System.out.println("Opción 3");
                break;
            default:
                System.out.println("Opción inválida");
        }
    }
    }
